src/components/phase-banner/index.md

```diff
@@ -40,10 +40,12 @@ See the full list of [components and patterns affected by WCAG 2.2](/accessibili
 
 Services hosted on a service.gov.uk domain must use the phase banner until they pass a live assessment.
 
-Use an alpha banner when your service is in alpha, and a beta banner if your service is in private or public beta.
-
 ## How it works
 
+There are 2 ways to use the phase banner component. You can use HTML or, if you are using [Nunjucks](https://mozilla.github.io/nunjucks/) or the [GOV.UK Prototype Kit](https://prototype-kit.service.gov.uk), you can use the Nunjucks macro.
```

Comment:
```suggestion
There are 2 ways to use the phase banner component. You can use:

- HTML
- Nunjucks macros – if you are using [Nunjucks](https://mozilla.github.io/nunjucks/) or the [GOV.UK Prototype Kit](https://prototype-kit.service.gov.uk)

```

Reply:
Mind if we look at this paragraph holistically? We've already got a couple of different versions (linked, unlinked) and we're not including it on every component page, so I'm reluctant to add a third variation…

Reply:
changing this around puts the 2 ways up front, then follows with the caveats.

Reply:
oh, yes if you like